src/client/datascience/serviceRegistry.ts

```diff
@@ -196,17 +196,23 @@ import { INotebookWatcher, IVariableViewProvider } from './variablesView/types';
 import { VariableViewActivationService } from './variablesView/variableViewActivationService';
 import { VariableViewProvider } from './variablesView/variableViewProvider';
 import { WebviewExtensibility } from './webviewExtensibility';
+import { IApplicationEnvironment } from '../common/application/types';
 
 // README: Did you make sure "dataScienceIocContainer.ts" has also been updated appropriately?
 
 // eslint-disable-next-line
 export function registerTypes(serviceManager: IServiceManager, inNotebookApiExperiment: boolean, inCustomEditorApiExperiment: boolean) {
-    const usingCustomEditor = inCustomEditorApiExperiment && !vscode.env.appName.includes('Insider'); // Don't use app manager in case it's not available yet.
+    const isVSCInsiders = serviceManager.get<IApplicationEnvironment>(IApplicationEnvironment).channel === 'insiders';
+    const usingCustomEditor = inCustomEditorApiExperiment && !isVSCInsiders; // Don't use app manager in case it's not available yet.

```

Comment:
I assume this comment doesn't apply anymore? You're using the application environment here.

Reply:
> Don't use app manager in case it's not available yet.

Yes.
Will remove that